apps/table-frontend/src/app/components/effective.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import { getFile } from './get-file-info';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+export interface EffectiveProps {
+  fileName: string;
+}
+
+interface Effectivity {
+  effective: string;
+  confidence: number;
+}
+
+const UNSURE: Effectivity = {
+  effective: 'unsure \\_(-_-)_/',
+  confidence: 1,
+};
+
+function checkIsEffective(
+  parsedParagraphs: ParsedArticleParagraphStandalone[]
+): Effectivity {
+  const totalRatedEffective = parsedParagraphs.reduce(
+    (totalEffective, parsedP) =>
+      (totalEffective +=
+        parsedP.backsUpClaim === ArticleParagraphBacksUpClaim.yes ? 1 : 0),
+    0
+  );
+  const totalRatedIneffective = parsedParagraphs.reduce(
+    (totalEffective, parsedP) =>
+      (totalEffective +=
+        parsedP.backsUpClaim === ArticleParagraphBacksUpClaim.no ? 1 : 0),
+    0
+  );
+  if (totalRatedEffective > parsedParagraphs.length * 0.5)
+    return {
+      effective: 'yes!',
```

Comment:
Can you declare these 'yes', 'no', 'unsure' strings in a const or enum to be used by the whole file?

Reply:
Added an enum